index.html

```diff
@@ -5151,6 +5160,214 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
 
   <p>See <a href="#language-maps" class="sectionRef"></a> for a description
     of using <a>language maps</a> to set the language of mapped values.</p>
+
+  <section class="changed informative"><h3>Base Direction</h3>
+    <p>It is also possible to annotate a <a>string</a>, or <a>language-tagged string</a>,
+      with its <a>base direction</a>.
+      As with language, it is possible to define a <a>default base direction</a> for a JSON-LD document
+      by setting the `@direction` key in the <a>context</a>:</p>
+
+  <aside class="example ds-selector-tabs changed"
+         title="Setting the default base direction of a JSON-LD document">
+    <div class="selectors">
+      <button class="selected" data-selects="compacted">Compacted (Input)</button>
+      <button data-selects="expanded">Expanded (Result)</button>
+      <button data-selects="statements">Statements</button>
+      <button data-selects="turtle">Turtle (drops direction)</button>
+      <button data-selects="turtle-dt">Turtle (with datatype)</button>
+      <button data-selects="turtle-bn">Turtle (with bnode structure)</button>
+      <a class="playground" target="_blank"></a>
+    </div>
+    <pre class="compacted input selected nohighlight" data-transform="updateExample">
+    <!--
+    {
+      "@context": {
+        "title": "http://example.org/title",
+        "publisher": "http://example.org/publisher",
+        ####...####
+        ****"@language": "ar",
+        "@direction": "rtl"****
+      },
+      ****"title": "HTML و CSS: تصميم و إنشاء مواقع الويب"****,
```

Comment:
```suggestion
      ****"title": "HTML و CSS: تصميم و إنشاء مواقع الويب",
```
(the closing 4-stars are present on the next line)